src/styles/variables.scss

```diff
@@ -0,0 +1,15 @@
+$widthOfDay: 100px;
+$heigthOfDay: $widthOfDay;
+$mgBtwnDays: 0.5px;
```

Comment:
The variable `$mgBtwnDays` is set to `0.5px`, which is an unusual value for a margin. Typically, margins are set to whole numbers for better rendering across different browsers.